stratum/hal/lib/p4/p4_info_manager.h

```diff
@@ -106,6 +106,10 @@ class P4InfoManager {
       const std::string& register_name) const;
   virtual ::util::StatusOr<const std::string> FindResourceTypeByID(
       uint32 id_key) const;
+  virtual ::util::StatusOr<const ::p4::config::v1::DirectPacketModMeter>
```

Comment:
Please add comment that these functions are for ES2K

Reply:
The files in hal/lib/p4 are platform-neutral. These methods should be for any meter that implements the underlying RFC. ES2K happens to be the first implementation.

Reply:
The new RFC changes didn't happen in p4lang p4runtime directly and we made the changes in es2k specific compiler. This would have made sense to keep in the common TDI layer if we had made the changes in p4runtime for all architectures/targets. GenericTables will make its way in p4lang common repo so the changes for that in TDI common layer in stratum would make sense. That's why for this change, I was proposing to create a derived class from the TDI common layer like it was done for PortManager